#Service Request
string stringInput
---
# Service Response
string stringOutput
